### PR TITLE
Add party to candidate model

### PR DESCRIPTION
**Pull request by ttavenner**

### CODE REVIEW

File: config/dev-fixtures/candidate-fixtures.js [hunk 1]
```diff
@@ -1,41 +1,43 @@
...
-        .then(offices => {
+    load: (app) => {
+      return app.orm.Candidate.count({})
+      .then(count => {
+          if (count > 0) {
+              return [];
+          }
+
+          const maxId = Math.max.apply(Math,candidates.map(function(o){return o.id;}));
+          app.orm.Survey.sequelize.query('select setval(\'candidate_id_seq\', ' + maxId + ')');
```
Comment (qwo) [inline config/dev-fixtures/candidate-fixtures.js:33]: so we need to escape if you used js template strings instead?

File: config/dev-fixtures/candidate-fixtures.js [hunk 2]
```diff
@@ -1,41 +1,43 @@
...
-        return app.orm.Office.findAll({where: {}})
-        .then(offices => {
+    load: (app) => {
+      return app.orm.Candidate.count({})
+      .then(count => {
+          if (count > 0) {
+              return [];
+          }
+
+          const maxId = Math.max.apply(Math,candidates.map(function(o){return o.id;}));
```
Comment (qwo) [inline config/dev-fixtures/candidate-fixtures.js:32]: did we want to use a short arrow func for this or do we want to preserve the context and have it s own scope?

File: config/dev-fixtures/candidate-fixtures.js [hunk 3]
```diff
@@ -1,41 +1,43 @@
...
+    {
+        id: 1,
+        name: 'Candidate A',
+        party: 'Republocrat',
+        OfficeId: 1
+    },
+    {
+        id: 2,
+        name: 'Candidate B',
+        party: 'Democan',
```
Comment (qwo) [inline config/dev-fixtures/candidate-fixtures.js:13]: Nice

File: config/dev-fixtures/candidate-fixtures.js (hunk 1 above)
Comment (ttavenner) [reply config/dev-fixtures/candidate-fixtures.js:33]: This same line is used in many fixture files. I will put in a separate PR to correct it.

File: config/dev-fixtures/candidate-fixtures.js (hunk 2 above)
Comment (ttavenner) [reply config/dev-fixtures/candidate-fixtures.js:32]: I don't think it makes much difference. This same line exists in many of the fixture files so if we want to do it differently I will put in a PR to fix it later.

**Review by wbprice — APPROVED** [review]

**Pull request merged**